src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -73,20 +73,22 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
     return <BaseSystemChat item={item} index={index} />;
   }
 
-  if (item?.isMe)
+  if (item?.isMe) {
     return (
       <ChatItemMyTextV2
         key={index}
         avatar={handleAvatar()}
         isContinuous={item?.isContinuous}
         message={item?.message}
+        attachments={item?.attachmentJson}
         time={calculateTime(item?.updatedAt, true)}
         username={handleUserName(item)}
         type={BaseChatItemTypeProps.MY_ANON_CHAT}
         status={item?.status as ChatStatus}
         chatType={type}
       />
     );
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces the `attachments` prop to the `ChatItemMyTextV2` component. However, it's not clear how the `attachmentJson` is parsed and used in the `ChatItemMyTextV2` component. If the `attachmentJson` is a JSON string, it should be parsed before being passed as a prop. Also, there should be error handling for parsing errors.

```diff
- attachments={item?.attachmentJson}
+ attachments={JSON.parse(item?.attachmentJson || '[]')}
```

Please ensure that the `ChatItemMyTextV2` component can handle the `attachments` prop correctly, including when it's an empty array or null.


<!-- This is an auto-generated comment by OSS CodeRabbit -->